src/Loop.php

```diff
@@ -223,6 +223,15 @@ public static function supports($feature) {
     }
 
     /**
+     * Get the underlying loop handle.
+     *
+     * @return null|object|resource The loop handle the event loop operates on. Null if there is none.
+     */
+    public static function getLoopHandle() {
```

Comment:
We already have PSR 1 and 2, and I don't have good enough reason to *not* follow it. It's just code style which isn't that significant, as long as there's some consistency. I follow PSR 1 and 2 everywhere, just because sharing code styles is better than arguing about which one is the best.